### ISSUE TITLE
test failure in chapter 5 

### ISSUE THREAD

**Issue by soonhokong** [leanprover/tutorial#38]
Dear @leodemoura  and @avigad ,

As you may notice, we have a test failure in chapter 5 of the tutorial. Here is the log:

```
-- testing 05_Interacting_with_Lean.org
code fragment #1 worked
code fragment #2 worked
code fragment #3 worked
code fragment #4 worked
code fragment #5 worked
code fragment #6 worked
code fragment #7 worked
code fragment #8 worked
code fragment #9 worked
code fragment #10 worked
code fragment #11 worked
code fragment #12 worked
ERROR executing 05_Interacting_with_Lean.org.13.lean, for in_code_block block starting at 515, produced output:
05_Interacting_with_Lean.org.13.lean:3:0: warning: imported file uses 'sorry'
05_Interacting_with_Lean.org.13.lean:6:10: error: invalid notation declaration, symbol expected
05_Interacting_with_Lean.org.13.lean:6:15: error: invalid local/reserve notation, 'infix', 'infixl', 'infixr', 'prefix', 'postfix' or 'notation' expected
05_Interacting_with_Lean.org.13.lean:7:7: error: invalid notation declaration, ':=' expected
05_Interacting_with_Lean.org.13.lean:7:12: error: invalid local/reserve notation, 'infix', 'infixl', 'infixr', 'prefix', 'postfix' or 'notation' expected
```

You can also find this failure at travis-ci: https://travis-ci.org/leanprover/tutorial/jobs/48438890#L3664-L3683

Note that we're using lean-0.2.0.20150126211556.git2a5658ebe28ad3f7b3be13fdac6213a1a4cf000d~12.04 which is the latest version (https://github.com/leanprover/lean/commit/2a5658ebe28ad3f7b3be13fdac6213a1a4cf000d) as I write this issue.

**Comment by avigad** [comment]
I am fixing this now...

**Comment by avigad** [comment]
These have to do with a recent change to the attribute declarations; now
persistent is the default, and we use "local attribute" for a temporary
change. I just issued a pull request, 7b8652e.

Later today, I will make a lot of small additions and changes to the
tutorial, and review these changes more carefully.

**Comment by soonhokong** [comment]
Closed by 7b8652e0a049c9931886ff2684d992821e6b7687
All tests passed at https://travis-ci.org/leanprover/tutorial/jobs/48493104

> Later today, I will make a lot of small additions and changes to the tutorial, and review these changes more carefully.

Thank you! Just as a reminder, you can run `make test` to check all the code in `.org` files are running without an error.
